etcdutl/snapshot/util.go

```diff
@@ -20,12 +20,24 @@ import (
 
 type revision struct {
 	main int64
+	sep  byte
```

Comment:
Why add `sep`?  Not needed

Reply:
This struct should be kept synched with revision from https://github.com/etcd-io/etcd/blob/main/server/storage/mvcc/revision.go